scripts/write_tip3p.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict
+
+import pandas
+from openff.toolkit.typing.engines.smirnoff.forcefield import ForceField
+from openff.toolkit.typing.engines.smirnoff.parameters import (
+    ConstraintHandler,
+    ElectrostaticsHandler,
+    LibraryChargeHandler,
+    vdWHandler,
+)
+from openff.units import unit
+from packaging import version
+
+VERSION = version.Version("1.0.0")
+OFFXML_PATH = "openforcefields/offxml/"
+
+dataframe = pandas.read_csv("openforcefields/data/jc.csv")
+
+tip3p = ForceField()
+
+tip3p_electrostatics = ElectrostaticsHandler(version=0.4)
+tip3p_library = LibraryChargeHandler(version=0.3)
+tip3p_vdw = vdWHandler(version=0.3)
+tip3p_constraints = ConstraintHandler(version=0.3)
+
+tip3p_vdw.add_parameter(
+    {
+        "smirks": "[#1]-[#8X2H2+0:1]-[#1]",
+        "epsilon": unit.Quantity(0.1521, unit.kilocalorie_per_mole),
+        "sigma": unit.Quantity(3.1507, unit.angstrom),
+        "id": "n-tip3p-O",
+    }
+)
+tip3p_vdw.add_parameter(
+    {
+        "smirks": "[#1:1]-[#8X2H2+0]-[#1]",
+        "epsilon": unit.Quantity(0.0, unit.kilocalorie_per_mole),
+        "sigma": unit.Quantity(1.0, unit.nanometer),
+        "id": "n-tip3p-H",
+    }
+)
+
+tip3p_library.add_parameter(
+    {
+        "smirks": "[#1]-[#8X2H2+0:1]-[#1]",
+        "charge1": unit.Quantity(-0.834, unit.elementary_charge),
+        "id": "q-tip3p-O",
+    }
+)
+tip3p_library.add_parameter(
+    {
+        "smirks": "[#1:1]-[#8X2H2+0]-[#1]",
+        "charge1": unit.Quantity(0.417, unit.elementary_charge),
+        "id": "q-tip3p-H",
+    }
+)
+
+element_to_atomic_number: Dict[str, int] = {
+    "Li": 3,
+    "Na": 11,
+    "K": 19,
+    "Rb": 37,
+    "Cs": 55,
+    "F": 9,
+    "Cl": 17,
+    "Br": 35,
+    "I": 53,
+}
+
```

Comment:
Please don't rubber-stamp these lines; I may have a PhD but I'm liable to misread and/or mistype short strings and ints.

Reply:
Checked and verified!